.github/workflows/multi-node-test-workflow.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         with:
           java-version: 14
       - name: Run integration tests with multi node config
-        run: ./gradlew integTest -PnumNodes=3 -Dopensearch.version=1.3.0-SNAPSHOT
```

Comment:
`main` is currently on version `1.3.0` so you'll want to develop the changes against that since this will be merged into `main`